templates/ansible/roles/redis/tasks/main.yml

```diff
@@ -14,3 +14,7 @@
 - name: Configure redis-rb
   template: src=redis.rb.j2 dest={{ app_config_path }}/initializers/redis.rb owner={{ user_name }} backup=yes
   when: redis_orm is defined and redis_orm == 'redis-rb'
+ 
+- name: Create upstart config
+  template: src=upstart.conf.j2 dest=/etc/init/redis.conf
+  notify: restart redis
```

Comment:
I believe `redis-server` Ubuntu package should create `init.d` script for starting redis. Will it conflict with upstart?